### ISSUE TITLE
Separate repeating code blocks into their own function

### ISSUE THREAD

**Issue by Tjzabel** [RITlug/teleirc#337]
# Summary

The `handler.go` Telegram file contains many repeating code blocks while checking for ZWSP. This should get moved into its own helper function in `helpers.go`. 

# Details

Example code block is as follows:
```go
username := ""
	if tg.IRCSettings.ShowZWSP {
		username = ZwspUsername(u.Message.From)
	} else {
		username = GetUsername(u.Message.From)
	}
```

# Outcome

The above code block is moved into its own separate function to reduce repeating code.

**Comment by nanikjava** [comment]
Happy to take this on. Thanks.

**Comment by Amirhossein2000** [comment]
Hi
Please review my pull request and fell free about any opinion.

**Comment by jwflory** [comment]
Thanks @Amirhossein2000, another core developer will finish reviewing PR #338.

@nanikjava, there are a few other "Hacktoberfest" good first issues to work on. Try taking a look at one of those and let us know if any of them look interesting to you! :slightly_smiling_face:

**Comment by Tjzabel** [comment]
Fixed via #338
